app/views/administrate/application/_search.html.erb

```diff
@@ -1,6 +1,8 @@
 <form class="search" role="search">
   <span class="search__icon">
-    <%= svg_tag "administrate/search.svg", "search", width: 16, height: 16 %>
+    <svg class="search__clear-icon">
+      <use xlink:href="#icon-eyeglass" />
```

Comment:
Within the state of this PR, this SVG should have a `aria-hidden` attribute, because it is presentational. I’ll add that, even though it gets refactored in #881.